Network/Public/Include/Network/iAPI.hpp

```diff
@@ -0,0 +1,184 @@
+#include <memory>
+
+#include "Network/Connection.hpp"
+#include "Network/Message.hpp"
+#include "Network/SafeQueue.hpp"
+
+using ClientConn = const std::shared_ptr<Network::Connection>&;
+
+#define APPLY_API_METHODS ;\
+            void messageClient                  (std::shared_ptr<Network::Connection> clientConnection,   const Message& message ) const override;   \
+            void messageAllClients              (std::shared_ptr<Network::Connection> exceptionConnection,const Message& message) const override;   \
+            void checkServerPing                (std::shared_ptr<Network::Connection> clientConnection,   const Message& message ) const override;   \
+            void readAllMessage                 (std::shared_ptr<Network::Connection> clientConnection,   const Message& message ) const override;   \
+            void channelListRequest             (std::shared_ptr<Network::Connection> clientConnection                           ) const override;   \
```

Comment:
Fix please this bracket

Reply:
What do you mean?

Reply:
Too much indentation from method name and sloppy inside arguments. We're fighting for the beauty of the code, aren't we?)

Reply:
What you are looking for? This is macros that should be as much as possible clear. So in this case, we could violate by styleguide